docs_guide/sphinx_guide/add_sections.rst

```diff
@@ -2,6 +2,8 @@
 Add sections to documentation
```

Comment:
Should this title should be changed to "How to add sections to documentation" like you did for other how-tos?

Reply:
Done in 129b3e07c368669d761a978193d8f4f63f250c22